app/population/view/dashboard.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "layouts/collection.njk" %}
+
+{% block pageContent %}
+  {% if resultsAsPopulationTable | length  %}
+    <div class="govuk-grid-row">
+      <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
+        <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
+          {{ t("dashboard::population.heading") }}
+
+          <span class="govuk-!-display-block">
+              {{ currentWeek | formatDateRangeWithRelativeWeek }}
```

Comment:
These two lines still look indented a little too much? This should fix this (and an extra space on line 4) up:

```nunjucks
{% extends "layouts/collection.njk" %}

{% block pageContent %}
  {% if resultsAsPopulationTable | length %}
    <div class="govuk-grid-row">
      <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
        <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
          {{ t("dashboard::population.heading") }}

          <span class="govuk-!-display-block">
            {{ currentWeek | formatDateRangeWithRelativeWeek }}
          </span>
        </h2>

        {{ govukTable(resultsAsPopulationTable) }}
      </section>
    </div>
  {% endif %}
{% endblock %}
```